build.gradle

```diff
@@ -3,6 +3,7 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```

Comment:
`google()` should go above `jcenter()`. Google for some reason has left their library listings in jcenter but taken the `aar` files out so it confuses the build system if jcenter is first.